src/app/features/sidebar/plus-button.tsx

```diff
@@ -72,7 +72,7 @@ export default function PlusButton() {
   }
   return (
     <>
-      <Button onClick={() => onClick()}>
+      <Button role="create" onClick={() => onClick()}>
```

Comment:
I don't think "create" is a valid ARIA Role. Is it?  https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles

You can do aria-label="create", that might be better.